docs/troubleshoot-selinux.md

```diff
@@ -0,0 +1,144 @@
+| `"/path/to/custom/data(/.*)?"` | Path to the custom data directory in the Server setup. The `(/.*)?` part is a regular expression pattern matching any files or subdirectories within the specified directory. |
+
+This command tells SELinux to label all files and subdirectories within the `/path/to/custom/data` directory with the SELinux type context `mysqld_db_t`. This operation ensures that SELinux treats these files and directories as part of the server's database, allowing the server to access them according to its SELinux policy.
+   
+### Use restorecon
+
+
+```{.bash data-prompt="$"}
+$ restorecon -Rv /path/to/custom/data
+```
+
+| Option | Description                                                                                          |
+|--------|------------------------------------------------------------------------------------------------------|
+| -R     | Recursively restore the SELinux context for the specified directory and all of its subdirectories.  |
+| -v     | Verbosely display the actions performed by the `restorecon` command, providing detailed feedback.    |
+
+The `restorecon -Rv /path/to/custom/data` command is used to restore the SELinux context for a specific directory and all of its subdirectories. Here's what each part of the command does:
+
+- `restorecon`: This is the main command used to restore the SELinux context of files and directories.
+- `-R`: This option stands for "recursive" and indicates that the command should operate recursively on all files and subdirectories within the specified directory.
+- `-v`: This option stands for "verbose" and instructs the command to display detailed information about the actions it performs, providing feedback on which files and directories had their SELinux context restored.
+
+The `/path/to/custom/data` part of the command should be replaced with the actual path to the directory for which you want to restore the SELinux context.
+
+As for whether this command requires a system reboot, typically, `restorecon` does not require a system reboot. It simply restores the SELinux context for the specified directory and its contents. However, if you're experiencing issues with SELinux after running the command, a system reboot may be necessary to ensure that all changes take effect.
+
+## Setting Custom Log Location
+
+When setting a custom log location for the server, SELinux permissions may need adjustment to allow the server to write to the new directory.
+
+### Example:
+
+1. Update SELinux tags:
+
+   ```{.bash data-prompt="$"}
+   $ semanage fcontext -a -t var_log_t "/path/to/custom/logs(/.*)?"
+   ```
+
+   This command associates the `var_log_t` type with the custom log directory and its contents.
+
+2. Restore SELinux context:
+
+   ```{.bash data-prompt="$"}
+   $ restorecon -Rv /path/to/custom/logs
+   ```
+
+   This command restores SELinux contexts recursively for the custom log directory, ensuring proper permissions for the server to write logs.
```

Comment:
The numbered list is broken on the preview https://percona-server-inno-release-pr-322.onrender.com/troubleshoot-selinux.html#example